packages/dev-server/src/middleware/JSInspectorMiddleware.ts

```diff
@@ -0,0 +1,127 @@
+import type { NextHandleFunction } from 'connect';
+import fs from 'fs-extra';
+import type { IncomingMessage, ServerResponse } from 'http';
+import fetch from 'node-fetch';
+import open from 'open';
+import path from 'path';
+import { TLSSocket } from 'tls';
+import { URL } from 'url';
+
+export default function JSInspectorMiddleware(): NextHandleFunction {
```

Comment:
I think this casing isn't correct, the other middleware is `clientLogsMiddleware` but maybe it should be `getClientLogsMiddleware`. Regardless, capitalize in JS often implies using the `new` keyword to instantiate it.